components/atoms/Search/search.tsx

```diff
@@ -154,7 +151,13 @@ const Search = ({
                   suggestion && handleOnSelect(suggestion);
                 }}
               >
-                {typeof suggestion === "string" ? <span className="pl-5 text-sm ">{suggestion}</span> : suggestion.node}
+                {typeof suggestion === "string" ? (
+                  <span className="pl-5 text-sm sm:max-w-[12.3rem] md:max-w-[11rem]md:mw-auto inline-block text-ellipsis truncate tracking-tighter">
```

Comment:
```suggestion
                  <span className="pl-5 text-sm md:mw-auto inline-block text-ellipsis truncate tracking-tighter">
```

Reply:
I've committed this locally after testing.